ipyleaflet/leaflet.py

```diff
@@ -1244,6 +1246,50 @@ def __init__(self, **kwargs):
         self.data = self._get_data()
 
 
+class WKTLayer(GeoJSON):
+    """WKTLayer class.
+
+    Layer created from a local WKT file or WKT string input.
+
+    Attributes
+    ----------
+    path: file path of local WKT file.
+    wkt_string: WKT string.
+    """
+
+    path = Unicode('')
+    wkt_string = Unicode('')
+
+    def __init__(self, **kwargs):
+        super(WKTLayer, self).__init__(**kwargs)
```

Comment:
Should we start using Python 3's syntax `super().__init__(**kwargs)`?

Reply:
done